model/model/v1alpha/model_public_service.proto

```diff
@@ -298,7 +334,16 @@ service ModelPublicService {
   // submitted as a binary file.
   rpc TriggerUserModelBinaryFileUpload(stream TriggerUserModelBinaryFileUploadRequest) returns (TriggerUserModelBinaryFileUploadResponse) {
```

Comment:
@heiruwu Is this endpoint used? I didn't see references in the backend.

Reply:
🆙

Reply:
yes its [here](https://github.com/instill-ai/model-backend/blob/main/pkg/handler/stream.go#L179), but it hasn't been used for awhile, and may be refactored to catch up with the design after pipeline streaming is ready.

Reply:
Right, I missed that reference. Then the header still applies there, though that entrypoint was missed in https://github.com/instill-ai/model-backend/pull/619/